autoload/db_ui/schemas.vim

```diff
@@ -23,11 +23,18 @@ let s:postgres_foreign_key_query = "
       \       ON ccu.constraint_name = tc.constraint_name
       \ WHERE constraint_type = 'FOREIGN KEY' and kcu.column_name = '{col_name}' LIMIT 1"
 
+let s:postgres_list_schema_query = "
+    \ SELECT nspname as schema_name
+    \ FROM pg_catalog.pg_namespace
+    \ WHERE schema_name !~ '^pg_'
```

Comment:
Why are you excluding schemas that start with `pg_`? Current query also returns those, so I don't want to break anything that's currently working.

Reply:
That is to avoid listing the `pg_temp_*` schemas that postgre creates, just in my production cluster is about 400 schemas so it can be annoying sometimes. A better solution could be to use `^pg_temp_` instead of just `^pg_`, to avoid filter those schemas. 
In my case for instance, the current query doesn't list this tables since I'm not the owner.

Reply:
I guess we could leave it for now with those, and introduce additional feature that would allow filtering schemas.

Reply:
That would be really nice. More info about this: 

https://postgrespro.com/list/thread-id/1374200

This Schemas are created when a temp table is created (we use them a lot), and a temp tables are only available within a session, so probably (thinking of a way to justify it) is of little interest to have the schemas listed if only the table is going to last the session time? 🤔

Reply:
I guess you are right. Use `^pg_temp_` as regex pattern and let's merge it. If anyone complains we'll put it back.